src/ActionList/List.tsx

```diff
@@ -47,25 +48,31 @@ export const List = React.forwardRef<HTMLUListElement, ActionListProps>(
       selectionVariant: containerSelectionVariant // TODO: Remove after DropdownMenu2 deprecation
     } = React.useContext(ActionListContainerContext)
 
+    const {containerRef} = useFocusZone({
+      bindKeys: FocusKeys.ArrowVertical | FocusKeys.HomeAndEnd | FocusKeys.PageUpDown
```

Comment:
Should the focus wrap when it hits the top and bottom?

```suggestion
      focusOutBehavior: 'wrap',
      bindKeys: FocusKeys.ArrowVertical | FocusKeys.HomeAndEnd | FocusKeys.PageUpDown
```

Reply:
You're right :) Thanks for the catch